components/public-api/gitpod/v1/workspace.proto

```diff
@@ -38,6 +43,55 @@ message WatchWorkspaceStatusResponse {
   WorkspaceStatus status = 2;
 }
 
+message ListWorkspacesRequest {
+  // pagination contains the pagination options for listing workspaces
+  PaginationRequest pagination = 1;
+
+  repeated WorkspaceSortRequest sorts = 2;
```

Comment:
I wonder whether `WorkspaceSortOption sort_options` will be better? But I'm not sure about the pattern here.

Reply:
also wonder whether it should be nested option SortOption inside ListWorkspacesRequest? Do you imagine it will be useful somewhere else?